csrankings-a.csv

```diff
@@ -1463,14 +1463,14 @@ Andrew C. Rice,University of Cambridge,https://www.cl.cam.ac.uk/~acr31,bxvOPw8AA
 Andrew C. Robb,Clemson University,http://arobb.people.clemson.edu,wkdrAV8AAAAJ
 Andrew C. Simpson,University of Oxford,http://www.cs.ox.ac.uk/andrew.simpson,rrydyHwAAAAJ
 Andrew C. Yao,Tsinghua University,http://iiis.tsinghua.edu.cn/yao,NOSCHOLARPAGE
-Andrew Calway,University of Bristol,http://www.bris.ac.uk/engineering/people/andrew-d-calway/index.html,zMXplXcAAAAJ
+Andrew Calway,University of Bristol,http://people.cs.bris.ac.uk/~andrew,jtlbdOoAAAAJ
 Andrew Chi-Chih Yao,Tsinghua University,http://iiis.tsinghua.edu.cn/yao,NOSCHOLARPAGE
 Andrew Cobley,University of Dundee,https://www.computing.dundee.ac.uk/about/staff/1,NOSCHOLARPAGE
 Andrew Coles,King's College London,https://www.kcl.ac.uk/people/andrew-coles,EMiUE40AAAAJ
 Andrew Colin Rice,University of Cambridge,https://www.cl.cam.ac.uk/~acr31,bxvOPw8AAAAJ
 Andrew Crabtree,University of Nottingham,https://www.nottingham.ac.uk/computerscience/people/andy.crabtree,LX_P5GYAAAAJ
-Andrew D. Miller 0001,IUPUI,https://soic.iupui.edu/people/andrew-miller,U5QeuUcAAAAJ
-Andrew D. Calway,University of Bristol,http://www.bris.ac.uk/engineering/people/andrew-d-calway/index.html,zMXplXcAAAAJ
+Andrew D Miller 0001,IUPUI,https://soic.iupui.edu/people/andrew-miller,U5QeuUcAAAAJ

```

Comment:
Restore period after middle initial (Andrew D. Miller).

Reply:
Apologies, fixed in 48553689210386a37f266ae5a89b2614f4239e22